external/avro/src/main/scala/org/apache/spark/sql/avro/AvroFileFormat.scala

```diff
@@ -130,7 +130,7 @@ private[avro] class AvroFileFormat extends FileFormat
 
   override def toString(): String = "Avro"
 
-  override def isSplitable(
+  override def isSplittable(
```

Comment:
haha. you know what? I believe this was from Hadoop (https://github.com/apache/hadoop/search?q=IsSplitable&unscoped_q=IsSplitable)

and although it's meant to be private, `FileFormat` one is pretty used here and there. Strictly we don't need to but might be good to keep the compatibility there.

Reply:
Yeah I recognize this typo from Hadoop. I think we should just leave it.

Reply:
+1 for just leaving it

Reply:
I just thought that Spark 3.0 would be suitable time to make such kind of changes in API otherwise the typo will stay here forever.

Reply:
hmm, this is breaking change every single spark app that calls this

Reply:
Apps won't call this directly, but they might extend the class. We can make breaking changes in Spark 3, but yes I'd leave this.